index.Rmd

```diff
@@ -19,14 +19,16 @@ library(bslib)
 [![Project Status](https://www.repostatus.org/badges/latest/wip.svg)](https://www.repostatus.org/#wip)
 [![CRAN status](https://www.r-pkg.org/badges/version/shinyMobile)](https://cran.r-project.org/package=shinyMobile)
 
+<br>
+
 ## Sandbox
```

Comment:
@DivadNojnarg currently the left panel navigation does not work because I had to make some JS changes. The input did not update. I think `rinterface-wasm-cran` needs to be updated to have these latest changes reflected? How does that work, just make a test somewhere so it gets redeployed?

Reply:
I added some notes from George regarding the package installation, which will retrigger a new deployment here: https://github.com/RinteRface/rinterface-wasm-cran/actions